english/explore-outside/journey-to-flaming-tower/start-running/keep-running/keep-running.md

```diff
@@ -2,3 +2,9 @@ You keep running toward the flaming tower.
 You run until your legs burn and your lungs feel like they will explode.  The presence behind you is getting closer.
 Closer with every step.  You feel hot breath on the back of your neck.
 
+A faint shadow emerges from between your feet. Warm gooey chocolate drips down your neck. "The tower! It's collapsing! It's melting chocolate!" you thought. Before you turn your attention back to your feet, your ankles are under thick chocolate goo.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.